consensus/dev/dev.go

```diff
@@ -150,6 +151,7 @@ func (d *Dev) writeNewBlock(parent *types.Header) error {
 
 	// Commit the changes
 	_, root := transition.Commit()
+	staking.GetStakingHub().ClearEvents()
```

Comment:
Please just add a comment that we're clearing events because they are added in the `transaction.Write()`, and need to be reset before we call `WriteBlocks()`

Reply:
Fixed 👍